src/command.spec.js

```diff
@@ -73,14 +73,38 @@ describe('#start()', () => {
         const command = new Command({ spawn: () => process });
 
         command.close.subscribe(data => {
-            expect(data).toBe('SIGKILL');
+            expect(data.exitCode).toBe('SIGKILL');
             done();
         });
 
         command.start();
         process.emit('close', null, 'SIGKILL');
     });
 
+    it('shares closes to the close stream with command info and index', done => {
+        const process = createProcess();
+        const command = new Command({
+            command: 'cmd',
+            name: 'name',
+            prefixColor: 'green',
+            env: { VAR: 'yes' },
+            index: 1,
+            spawn: () => process,
+        });
+
+        command.close.subscribe(data => {
+            expect(data.command).toBe('cmd');
+            expect(data.name).toBe('name');
+            expect(data.prefixColor).toBe('green');
+            expect(data.env).toEqual({ VAR: 'yes' });
+            expect(data.index).toBe(1);
```

Comment:
Can simplify to this with the above suggestion

```suggestion
            expect(data.command).toBe(command);
```